app/src/main/res/values/strings.xml

```diff
@@ -290,4 +290,7 @@
         <item quantity="one"><xliff:g>%1$d</xliff:g> card (<xliff:g id="archivedCount">%2$d</xliff:g> archived)</item>
         <item quantity="other"><xliff:g>%1$d</xliff:g> cards (<xliff:g id="archivedCount">%2$d</xliff:g> archived)</item>
     </plurals>
-</resources>
\ No newline at end of file
+    <string name="previousCard">Previous card</string>
```

Comment:
```suggestion
    <string name="previousCard">Previous</string>
```
Avoids "card" (where possible?)

Reply:
Hmm, maybe? I mean, this is only used as accessibility and long-press text. Would it be obvious to blind users what "Previous" and "Next" means when viewing a loyalty card? I guess it may be.